client/components/user/UserPrivacy.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <v-layout align-center>
+    <v-card class="card-container">
+      <h1 class="title">Change password</h1>
+      <form @submit.prevent="submit">
+        <v-text-field
+          v-validate="'required|alphanumerical|min:6'"
+          v-model="currentPassword"
+          :error-messages="vErrors.first('currentPassword')"
+          type="password"
+          data-vv-name="currentPassword"
+          label="Current password"/>
+        <v-text-field
+          v-validate="'required|alphanumerical|min:6'"
+          ref="newPassword"
+          v-model="newPassword"
+          :error-messages="vErrors.first('newPassword')"
+          type="password"
+          name="newPassword"
+          label="New password"/>
+        <v-text-field
+          v-validate="'required|confirmed:newPassword'"
+          v-model="reNewPassword"
+          :error-messages="vErrors.first('passwordConfirmation')"
+          data-vv-as="new password"
+          type="password"
+          name="passwordConfirmation"
+          label="Please re-enter your new password"/>
+        <v-btn :disabled="!isValid" type="submit" color="blue-grey darken-1" flat large>
+          Submit
+        </v-btn>
+      </form>
+    </v-card>
+  </v-layout>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex-module';
+import { withValidation } from 'utils/validation';
+
+export default {
+  mixins: [withValidation()],
+  data() {
+    return {
+      currentPassword: '',
+      newPassword: '',
+      reNewPassword: ''
+    };
+  },
+  computed: {
+    ...mapGetters(['user']),
+    isValid() {
+      return (this.currentPassword && this.newPassword && !this.vErrors.count());
```

Comment:
Why do you need checking for `currentPassword` and `newPassword` when they are marked as required?